### DISCUSSION TITLE
Is a negative pitch possible?

### DISCUSSION THREAD

**Discussion by wiesehahn** [visgl/deck.gl#8542] [General]
I would like to set a negative pitch value to kind of look further south from the orthogonal camera view. This seems not possible, or is it?

When trying this on codepen (https://codepen.io/ezekielchentnik/pen/eeNMKp) I recognized that the arclayer seems to be initialized with negative pitch but not the basemap, not sure if this is supposed to be like that or if its a bug?!

Pitch: 40 looks correct
![grafik](https://github.com/visgl/deck.gl/assets/41429613/e553b877-1dc5-4e71-bf93-99eeb0682748)

Pitch: -40 looks incorrect
![grafik](https://github.com/visgl/deck.gl/assets/41429613/7ffc016f-96bb-4b75-9b38-9c89e60aded1)

**Comment by Pessimistress** [comment]
You can totally use negative pitch in deck.gl.

The base map is rendered by mapbox/maplibre and they don't support negative pitch.
